pkg/resources/collector/workloads.go

```diff
@@ -22,9 +22,9 @@ const (
 	receiversKey       = "receivers.yaml"
 	mainKey            = "config.yaml"
 	aggregatorKey      = "aggregator.yaml"
-	collectorImageRepo = "ghcr.io/rancher-sandbox"
-	collectorImage     = "opni-otel-collector"
-	collectorVersion   = "v0.1.1-0.74.0"
+	collectorImageRepo = "docker.io/alex7285"
```

Comment:
Is this the correct repo?

Reply:
ah right need to release a new version, missed that

Reply:
https://github.com/rancher-sandbox/opni-otel-collector/pull/5, I know these PRs are overkill but it makes sense to me to document and sanity check changes to our otel distro